storage/innobase/include/buf0buf.h

```diff
@@ -2297,6 +2297,10 @@ struct buf_pool_t {
   running. Protected by flush_state_mutex. */
   os_event_t no_flush[BUF_FLUSH_N_TYPES];
 
+  /** This is event is set when there is no ongoing invalidation of the
+   * buff_pool_t. */
+  os_event_t no_invalidate;
```

Comment:
LRU threads are stopped by using this event. The user is "buf pool invalidate". Maybe we should rename it to stop_lru ? or pause_lru?

Reply:
Good point! Please, mind I have called the other way round that sugested: `buf_pool->run_lru`

Reply:
Looks good